apps/chat-e2e/src/assertions/manageAttachmentsAssertion.ts

```diff
@@ -36,4 +37,41 @@ export class ManageAttachmentsAssertion {
       .soft(arrowIconColor[0], ExpectedMessages.sharedIconColorIsValid)
       .toBe(expectedColor);
   }
+  public async assertEntityState(
+    entity: TreeEntity,
+    fileModalSection: FileModalSection,
+    expectedState: ElementState,
+  ) {
+    let entityTree: AttachFilesTree;
+    if (fileModalSection === FileModalSection.AllFiles) {
```

Comment:
what do you think about using `switch-case` and adding all available `FileModalSection` ?